base/logging.h

```diff
@@ -0,0 +1,1065 @@
+#pragma once
+
+#include "glog/logging.h"
+
+namespace logging {
+  constexpr int LOG_INFO = google::GLOG_INFO;
```

Comment:
we should see whether there's a better place to pull this from. We may get it transitively from gflags or gtest already, without needing Chromium's version. Not sure.

Also could be a follow-up.